### ISSUE TITLE
What should our default search query operator be?

### ISSUE THREAD

**Issue by richard-jones** [JiscMonitor/monitor-uk#34]
At the moment the default search operator on the search string is "OR".

This means if you search for 

```
An NMR crystallography study of the hemihydrate
```

you are actually searching

```
An OR NMR OR crystallography OR study OR of OR the OR hemihydrate
```

The alternative is to set it to AND

There are pros and cons:

AND:
- Gives you fewer, more tightly scoped results.  If you know what you're looking for, you will find it easily
- Does not allow for fuzziness or mistakes.  You get one word wrong, you don't find the answer

OR:
- Gives you a /lot/ of results, which means what you expect to see might not be there
- Allows for you to make mistakes and still find what you were looking for

If the results are ordered by relevance (one of the sort options), then OR queries will still return best results first, so another alternative would be to switch the default sort back to that (currently set as Most Recent).

@TomYeahOK - assigning to you for initial thoughts.  Mine would be to leave this as-is, I think, but I'm open to change.

**Comment by richard-jones** [comment]
Note as well that if you really want to search for an exact string, you can put it in quotes:

```
"An NMR crystallography study of the hemihydrate"
```

**Comment by tdcodesthings** [comment]
I agree with your reasoning, and feel that we should make a note to include something to clarify/advise users that quotes and other operators can be used.

**Comment by richard-jones** [comment]
@TomYeahOK - do you want to do anything about this before the beta release.  If so, let me know, and I can add some text to the page somewhere.  Otherwise, I'll just leave this issue here as a reminder to revisit in the future.

**Comment by richard-jones** [comment]
We're out of time now for the first production release to do any more, but I don't think any particular action is required at this point anyway.  Will leave this open but bump it from the milestone
